spec/feature/authenticated_host_spec.rb

```diff
@@ -54,4 +54,41 @@
     expect(page).to have_content("Jane's Dashboard")
   end
 
+  it "can visit its host dashboard and see their dashboard links" do
+    allow_any_instance_of(ApplicationController).to receive(:current_user).
+      and_return(host_user)
+    visit root_path
+    click_link_or_button("Dashboard")
+    expect(page).to have_content("Edit Your Listings")
+    expect(page).to have_content("Manage Reservations")
+    expect(page).to have_content("Reservation Requests")
+  end
+
+  it "can view their listings as links to the edit page" do
+    valid_user = create(:user, role: 1)
+    listing = create(:listing)
```

Comment:
yeah I just added role to factory girl and then changed it to 1 in the test to make it a host. I was having some issues when creating a user and a host as it was saying i had repeated email and display name so I had to change those too. If I did it wrong Im more than happy to change it.